README.md

```diff
@@ -61,6 +61,27 @@ The release process is building all runtimes and then puts them into a release i
 
 The format of [`CHANGELOG.md`](CHANGELOG.md) is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
+## Release guidelines
+
+Here are the recommended steps for both major and minor runtime releases.
+
+|Steps |Description |Major |Minor |
+|------|------------|------|------|
+|0 |Open an [issue](https://github.com/polkadot-fellows/runtimes/issues) for the release in the runtimes repo with the **version number**. | ✔️ | ✔️ |
```

Comment:
I have never started a release process myself to be able to say if these guidelines resemble the actual steps, but in general having them here is good. 

If the guidelines are well written, this will allow more people to take ownership of triggering the release, this should be end goal.

Reply:
> I have never started a release process myself to be able to say if these guidelines resemble the actual steps, but in general having them here is good.
> 
> If the guidelines are well written, this will allow more people to take ownership of triggering the release, this should be end goal.

Thank you for the feedback.
I have been working on these guidelines for the past 2 months with Bastian and Joe doing early reviews.  However, I still need to get more eyes on this before it can be merged into the Runtimes repo.